### PR TITLE
[fix](planner)fix bug of process sequence column in insert statement

### PR DESCRIPTION
**Pull request by starocean999**
## Proposed changes

```
CREATE TABLE `dws_ncc_detail_xcb_total_5` (
  `code` varchar(50) NULL COMMENT '编码',
  `value` float NULL,
  `ct` datetime NULL DEFAULT CURRENT_TIMESTAMP
) ENGINE=OLAP
UNIQUE KEY(`code`)
COMMENT 'OLAP'
DISTRIBUTED BY HASH(`code`) BUCKETS 8
PROPERTIES (
"replication_num" = "1",
"in_memory" = "false",
"storage_format" = "V2",
"function_column.sequence_col" = "ct",
"disable_auto_compaction" = "false"
);

insert into dws_ncc_detail_xcb_total_5 (code,value) values("a",1.22); 

org.apache.doris.common.AnalysisException: errCode = 2, detailMessage = Unexpected exception: Failed analysis after expr substitution: null
        at org.apache.doris.qe.StmtExecutor.analyze(StmtExecutor.java:750) ~[doris-fe.jar:1.2-SNAPSHOT]
        at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:446) ~[doris-fe.jar:1.2-SNAPSHOT]
        at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:409) ~[doris-fe.jar:1.2-SNAPSHOT]
        at org.apache.doris.qe.ConnectProcessor.handleQuery(ConnectProcessor.java:333) ~[doris-fe.jar:1.2-SNAPSHOT]
        at org.apache.doris.qe.ConnectProcessor.dispatch(ConnectProcessor.java:476) ~[doris-fe.jar:1.2-SNAPSHOT]
        at org.apache.doris.qe.ConnectProcessor.processOnce(ConnectProcessor.java:703) ~[doris-fe.jar:1.2-SNAPSHOT]
        at org.apache.doris.mysql.ReadListener.lambda$handleEvent$0(ReadListener.java:52) ~[doris-fe.jar:1.2-SNAPSHOT]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_333]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_333]
        at java.lang.Thread.run(Thread.java:750) ~[?:1.8.0_333]
```
this pr fix this problem

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by zhannngchen — COMMENTED** [review]
please also add some cases

**Comment by starocean999** [comment]
> please also add some cases

case added

File: regression-test/suites/correctness_p0/test_insert_sequence_column.groovy [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        "replication_num" = "1",
+        "in_memory" = "false",
+        "storage_format" = "V2",
+        "function_column.sequence_col" = "ct",
+        "disable_auto_compaction" = "false"
+        );
+    """
+
+    sql """
+        insert into dws_ncc_detail_xcb_total_5 (code,value) values("a",1.22);
```
Comment (zhannngchen) [inline regression-test/suites/correctness_p0/test_insert_sequence_column.groovy:42]: you'd better to add some check for the data you inserted, make sure it works well, thanks

Comment (starocean999) [reply regression-test/suites/correctness_p0/test_insert_sequence_column.groovy]: done

**Review by zhannngchen — APPROVED** [review]
LGTM

**Pull request merged**